### DISCUSSION TITLE
Add Silk.NET to GLFW community page

### DISCUSSION THREAD

**Discussion by blair-ahlquist** [dotnet/Silk.NET#1642] [Q&A]
Should Silk.NET be listed on this page? https://www.glfw.org/community.html#bindings

**Comment by Perksey** [comment]
Yes probably. Not our decision to make though, if someone wants to add it we won’t object :)

**Comment by blair-ahlquist** [comment]
Ok. Added an issue on their repo: https://github.com/glfw/glfw.github.io/issues/97
